kubernetes-proxy/config.json.template

```diff
@@ -6,10 +6,7 @@
     },
     "process": {
         "terminal": false,
-        "user": {
-            "uid": 994,
-            "gid": 996
-        },
+        "user": {},
```

Comment:
Other system images denote:

```json
"user": {
    "uid": 0,
    "gid": 0
},
```

Is leaving this empty fine?

Reply:
Leaving it out defaults to 0, but it's probably clearer to be explicit. I'll change it.